File: AnnexNoSQL/src/Mongo/Prova2.java
package Mongo;

import org.bson.Document;

import com.mongodb.MongoClient;
import com.mongodb.client.FindIterable;
import com.mongodb.client.MongoDatabase;

public class Prova2 {

    public static void main(String[] args) {
        MongoClient con = new MongoClient("localhost" , 27017);
        MongoDatabase bd = con.getDatabase("test");
        
        FindIterable<Document> llibres =  bd.getCollection("libro").find();
        
        for (Document llibre : llibres)
            System.out.println(llibre.get("titulo"));
        
        con.close();
    }
}

File: Tema8_2/src/exemples/Prova2.kt
package exemples

import com.mongodb.MongoClient

fun main(){
    val con = MongoClient("localhost" , 27017)
    val bd = con.getDatabase("test")
   val llibres = bd.getCollection("libro").find()

    for (llibre in llibres)
        println(llibre.get("titulo"))

   con.close();
}